examples/interaction/js_callbacks/customjs_for_hover.py

```diff
@@ -1,3 +1,15 @@
+''' An interactive plot showcasing Bokeh's ability to add interactions
+using Custom Javascript.
+This example demonstrates adding links between points on a graph.
+These lines only show on hover using ``CustomJS`` callbacks to update the plot.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.segment, bokeh.plotting.figure.circle, bokeh.plotting.figure.add_tools, bokeh.models.HoverTool, bokeh.models.sources.ColumnDataSource, bokeh.models.callbacks.CustomJS
+    :refs: :ref:`ug_interaction_js_callbacks_customjs`
+    :keywords: hover, javascript callback, CustomJS
+'''
```

Comment:
```suggestion

'''
```
blank line at the end